tools/build_pak_files.py

```diff
@@ -138,7 +138,11 @@
             assert os.path.exists(pak_file)
             print(f"[SPEAR | build_pak_files.py] Successfully built {pak_file} for {platform} platform.")
 
-            print(f"[SPEAR | build_pak_files.py] Removing symlink: {unreal_project_content_scene_dir}")
-            os.unlink(unreal_project_content_scene_dir)
+        print(f"[SPEAR | build_pak_files.py] Removing symlink: {unreal_project_content_scene_dir}")
```

Comment:
Either include an assert here like you do for the other symlink, or remove the other assert, for consistency.

Reply:
No need of assert here, removed it.